submissions/ogennaOgbogu/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should include this `.vscode` file to `.gitignore` so that you won't be committing it.